### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by Quantalabs**
Add nightly docs with `sphinx`, which will be published with [readthedocs](https://readthedocs.org) after merging.

### CODE REVIEW

File: .github/workflows/python-publish-nightly.yml [hunk 1]
```diff
@@ -38,7 +38,12 @@ jobs:
         flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
     - name: Change Version Info
       run: | 
-        python release.py
+        python scripts/release.py
```
Comment (quantum9Innovation) [inline .github/workflows/python-publish-nightly.yml:41]:
```suggestion
        python release.py
```
Moving `release.py` into the `scripts/` directory will limit its access to root files meaning that `setup.py` version info won't be able to be changed.

File: docs/conf.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+#
+import os
+import sys
+sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'epispot'
+copyright = '2021, Quantalabs, Quantum9Innovation'
```
Comment (quantum9Innovation) [inline docs/conf.py:21]:
```suggestion
copyright = '2021, @Quantalabs, @quantum9Innovation'
```

File: docs/conf.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = 'alabaster'
+
+# Add any paths that contain custom static files (such as style sheets) here,
+# relative to this directory. They are copied after the builtin static files,
+# so a file named "default.css" will overwrite the builtin "default.css".
+html_static_path = ['_static']
```
Comment (quantum9Innovation) [inline docs/conf.py:54]:
```suggestion
html_static_path = ['_static']

```
Add newline at EOF

File: epispot/__init__.py [hunk 4]
```diff
@@ -20,6 +20,6 @@
 from . import fitters
 
 # version info
-__version__ = "v2.0.1.22"  # version (v#.#.#)
+__version__ = "v2.0.1.23"  # version (v#.#.#)
```
Comment (quantum9Innovation) [inline epispot/__init__.py:23]: Doesn't match version from Sphinx
Choose one version and stick with it

File: scripts/docs.py [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+read_setup1 = open('../docs/conf.py', 'r').readlines()
+write_setup = open('../docs/conf.py', 'w')
+read_setup1[24] = 'release="'+version_info+'.'+str(commit_num + 1)+'"\n'
+ws = ''
+for x in read_setup1:
+    ws += x
+write_setup.write(ws)
+print('docs/conf.py release info updated')
+
+with os.scandir('../epispot/') as it:
```
Comment (quantum9Innovation) [inline scripts/docs.py:16]: Choose less generic variable name

File: scripts/docs.py [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+    for entry in it:
+        if entry.is_file() and not entry.name == '__init__.py' and not entry.name.endswith('.cpython-38.pyc'):
+            rstbase = entry.name.split('.')[0]+''' Module
+=====================================================================================
+.. automodule:: epispot.'''+entry.name.split('.')[0]+'''
```
Comment (quantum9Innovation) [inline scripts/docs.py:21]: :question:

File: scripts/docs.py [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+            f = open('../docs/'+entry.name.split('.')[0]+'.rst', 'w')
+            f.write(rstbase)
+
+print('Docs updated')
+
+os.system('cd docs')
+os.system('make html')
+
+print('Updated HTML')
+print('Job completed!')
```
Comment (quantum9Innovation) [inline scripts/docs.py:34]: Has this been tested?

**Review by quantum9Innovation — CHANGES_REQUESTED** [review]
Minor code quality issues, possible testing issues, and quick fixes

File: .github/workflows/python-publish-nightly.yml (hunk 1 above)
Comment (Quantalabs) [reply .github/workflows/python-publish-nightly.yml:41]: It doesn't make a difference in the script. I tested it on the computer myself.

File: scripts/docs.py (hunk 7 above)
Comment (Quantalabs) [reply scripts/docs.py:34]: Yes

File: scripts/docs.py (hunk 6 above)
Comment (quantum9Innovation) [reply scripts/docs.py:21]: What does this code _do_?

File: docs/conf.py [hunk 8]
```diff
@@ -51,4 +51,4 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
\ No newline at end of file
+html_static_path = ['_static']
```
Comment (quantum9Innovation) [inline docs/conf.py:54]:
```suggestion
html_static_path = ['_static']

```
Keep the newline!

**Pull request merged**